server/controllers/rest/update/SessionUpdateControllerREST.test.ts

```diff
@@ -0,0 +1,53 @@
+import SessionB2RConverter from "../../../presenters/rest/SessionB2RConverter";
+import UpdateSessionUseCase from "../../../use-cases/input-boundary-models/UpdateSessionUseCase";
+import SessionUpdateControllerREST from "./SessionUpdateControllerREST";
+import { mock } from "jest-mock-extended";
+import SessionOutputData from "../../../output-data/SessionOutputData";
+import { GameState } from "../../../entities/game-state/GameState";
+import { getMockReq, getMockRes } from "@jest-mock/express";
+import { Request, Response } from "express";
+
+describe("Session update controller", () => {
+  let controller: SessionUpdateControllerREST;
+
+  function mockInteractor() {
+    const interactor = mock<UpdateSessionUseCase>();
+    interactor.update.mockReturnValue(
+      new SessionOutputData("123", GameState.Running, ["a"], ["b", "c"])
+    );
+    return interactor;
+  }
+
+  function createController() {
+    controller = new SessionUpdateControllerREST(
+      mockInteractor(),
+      new SessionB2RConverter()
+    );
+  }
+
+  beforeEach(() => {
+    createController();
+  });
+
+  it("is created", () => {
```

Comment:
nesinorėtų pirmiau matyti faile testus, o tik po to helper funkcijas kur mockus kuria?